prep-javascript-dom-creation/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript DOM Creation</title>
+</head>
+
+<body>
+  <div id="container">
+  </div>
+  <script>
+ var $container = document.querySelector('container');
+ console.log($container);
```

Comment:
Actually i don't think this gives you what you want.

Reply:
Okay, I fixed it back and I think I committed the new file? I hope so

Reply:
<img width="442" alt="Screen Shot 2022-03-05 at 1 49 57 PM" src="https://user-images.githubusercontent.com/99847558/156901522-123d7fb2-c5ef-4daf-81f2-aaaf6a5ac3e9.png">